husky_components/datagrid/main.js

```diff
@@ -495,6 +495,24 @@
             },
 
             /**
+             * deselects an item with a given id
+             * @event husky.datagrid.deselect.item
+             * @param {String|Number} The id of the item to deselect
+             */
+            DESELECT_ITEM = function() {
+                return this.createEventName('deselect.item');
```

Comment:
Ok, then it is just different for all other components. But I agree, changing that would be a too big BC break.